File: js/task2b/S2_FOR_06.js
function sumNested(matrix) {
    let totalSum = 0;

    for (let row of matrix) {
        for (let num of row) {
            totalSum += num;
        }
    }

    return totalSum;
}

console.log(sumNested([[1, 2], [3, 4]]));         
console.log(sumNested([[1], [2, 3], [4, 5, 6]])); 
console.log(sumNested([]));                       
console.log(sumNested([[]]));                   

File: python/task2b/S2_FOR_06.py
def sumNested(matrix):
  total = 0
  for row in matrix:
      for num in row:
          total += num
  return total

print(sumNested([[1, 2], [3, 4]]))
print(sumNested([[1], [2, 3], [4, 5, 6]]))
print(sumNested([]))
print(sumNested([[]]))
